mev_inspect/classifiers/specs/zero_ex.py

```diff
@@ -1,9 +1,78 @@
+from typing import Optional, List
+from mev_inspect.schemas.transfers import Transfer
+from mev_inspect.schemas.swaps import Swap
 from mev_inspect.schemas.traces import (
+    DecodedCallTrace,
     Protocol,
 )
 from mev_inspect.schemas.classifiers import (
     ClassifierSpec,
+    SwapClassifier,
 )
+from mev_inspect.classifiers.helpers import (
+    get_amount_transferred_to_address,
+    get_amount_transferred_by_token_address,
+)
+
+
+class ZeroExSwapClassifier(SwapClassifier):
+    @staticmethod
+    def parse_swap(
+        trace: DecodedCallTrace,
+        prior_transfers: List[Transfer],
+        child_transfers: List[Transfer],
+    ) -> Optional[Swap]:
+
+        token_in_amount: int = 0
+        taker_address: str = ""
```

Comment:
use none instead of empty string for absence

then a string is always a valid value v. sometimes valid